src/org/sipdroid/sipua/PublishAgent.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright (C) 2016 Pranav Jain - The NorthCap University - India

```

Comment:
You don't need to mention your university in a Copyright notice unless the work is funded or otherwise assisted by the university